resources/tsconfig.build.cjs.json

```diff
@@ -14,6 +14,9 @@
     {
       "path": "../packages/codemirror-graphql"
     },
+    {
```

Comment:
Make sure to add to build.esm.json as well, and then we can remove rollup yeah?

Reply:
I've added an entry. However I'm not sure we can remove rollup just yet. It depends on a custom [codemirror build helper](https://github.com/codemirror/buildhelper) which uses rollup

Reply:
Well if you want the cross repo tsc —build and tsc —build —watch to work, you just need these and the matching tsconfig files which just extend the base files. Is the rollup required to bundle the plugin or all of codemirror? The bundle treats codemirror as an external yes

Reply:
Looking at the source code of the build helper, it depends on rollup. We could potentially have a custom build script without depending on the codemirror build helper and remove the need for rollup, but this is the path of least resistance.